plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
-import itertools
 import logging
```

Comment:
@omry 


In future PR/commits I want to change this file to `_core.py` to avoid import. but for now I will not rename so it's easier to review the diff.

Reply:
Ok.